thirdparty/spdlog/BUILD

```diff
@@ -0,0 +1,12 @@
+load("@rules_cc//cc:defs.bzl", "cc_library")
+
+cc_library(
+    name = "spdlog",
+    hdrs = glob([
+    	"include/**/*.cc",
+        "include/**/*.h",
```

Comment:
do "bazel run buildifier" from the root of this project, it'll format all these files